.github/workflows/centos_7.yml

```diff
@@ -59,13 +59,9 @@ jobs:
       - uses: ./.github/actions/environment
       - name: packaging
         env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          AWS_S3_ENDPOINT_URL: ${{ secrets.AWS_S3_ENDPOINT_URL }}
-          RELEASE_REPO_S3_DIR: ${{ secrets.RELEASE_REPO_S3_DIR }}
-          PRERELEASE_REPO_S3_DIR: ${{ secrets.PRERELEASE_REPO_S3_DIR }}
-          GPG_SECRET_KEY: ${{ secrets.GPG_SECRET_KEY }}
-          GPG_SIGN_KEY: ${{ secrets.GPG_SIGN_KEY }}
+          RWS_URL: ${{ secrets.RWS_URL }}
```

Comment:
NOT added RWS_URL into secrets. I would prefer explicit URL. It does not look as a private thing.

Reply:
Fixed